lib/opengl32-sys/src/lib.rs

```diff
@@ -11,8 +11,8 @@ extern "system" {
     // pub fn GlmfEndPlayback();
     // pub fn GlmfInitPlayback();
     // pub fn GlmfPlayGlsRecord();
-    // pub fn glAccum();
-    // pub fn glAlphaFunc();
+    pub fn glAccum(op: GLenum, value: GLfloat);
+    // pub fn glAlphaFunc(func: GLenum, ref: GLclampf);
```

Comment:
The parameter name has no effect on the functionality of the FFI binding. Feel free to rename it to something that is similar but isn't a keyword.